qutebrowser/browser/webkit/mhtml.py

```diff
@@ -138,6 +139,22 @@ def _check_rel(element):
     return any(rel in rels for rel in must_have)
 
 
+def _encode_quopri_mhtml(msg):
+    """Encode the message's payload in quoted-printable.
+
+       Substitute for quopri's default 'encode_quopri' method, which needlessly
+       encodes all spaces and tabs, instead of only those at the end on the
+       line.
```

Comment:
Dedent the description by one, as per [pep 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)